docs/samples/browser-plugin-meetings/index.html

```diff
@@ -104,13 +106,113 @@ <h2 class="collapsible">Meeting Management</h2>
 
               <form id="oauth" class="hidden">
                 <div>
-                  <button id="oauth-login-btn" type="submit">Login</button>
+                  <button id="oauth-login-btn" type="submit" class="btn-code">Login</button>
                   <p id="oauth-status" class="status-par">Not Logged In.</p>
                 </div>
               </form>
             </fieldset>
           </div>
 
+          <!-- Track / effects -->
+          <div>
+            <form id="tracks">
+              <fieldset>
+                <legend>Tracks</legend>
+                <div class="video-section">
+                  <div>
+                    <div class="flex" style="gap: 0.5rem;">
+                      <fieldset>
+                        <legend>Local Video</legend>
+                        <div class="video-container">
+                          <video id="local-video" class="video-el" muted="true" autoplay playsinline
+                            style="height: 250px;"></video>
+                          <span id="local-video-resolution" class="video-resolution"></span>
+                          <span id="local-video-published-state" class="video-published"></span>
+                        </div>
+                      </fieldset>
+
+                      <fieldset style="display: flex; align-items: center;">
+                        <legend>Local Audio</legend>
+                        <div class="audio-container">
+                          <audio id="local-audio" controls autoplay></audio>
+                          <span id="local-audio-published-state" class="audio-published"></span>
+                        </div>
+                      </fieldset>
+
+                    </div>
+                    <div class="flex">
+                      <!--enable virtual bg -->
+                      <div>
+                        <div class="u-mb">
+                          <button id="ts-load-camera" type="button" onclick="loadCamera()" class="btn-code">Load
+                            Camera</button>
+                        </div>
+                      </div>
+                      <div>
+                        <div class="u-mb">
+                          <button id="ts-enable-VBG" type="button" onclick="handleVbg()" class="btn-code">Enable
+                            VBG</button>
+                        </div>
+                      </div>
+                      <!--mute-unmute-video -->
+                      <div>
+                        <div class="u-mb">
+                          <button id="ts-toggle-video" type="button" onclick="toggleSendVideo()" class="btn-code">Toggle
+                            Video</button>
+                        </div>
+                      </div>
+                    </div>
+
+
+                    <div class="flex">
+                      <!--enable noise reduction -->
+                      <div>
```

Comment:
This seems like extra div serving no purpose. Similar occurrences are at multiple places